guides/source/layout.html.erb

```diff
@@ -39,59 +45,83 @@
         <li class="more-info"><a href="https://github.com/rails/rails">Contribute on GitHub</a></li>
       </ul>
     </div>
-  </div>
-  <div id="header">
+  </nav>
+  <header id="page_header">
     <div class="wrapper clearfix">
-      <div class="header-logo"><a href="index.html" title="Return to home page">Guides.rubyonrails.org</a></div>
-      <ul class="nav">
-        <li><a class="nav-item" href="index.html">Home</a></li>
-        <li class="guides-index guides-index-large">
-          <a href="index.html" id="guidesMenu" class="guides-index-item nav-item">Guides Index</a>
-          <div id="guides" class="clearfix" style="display: none;">
-            <hr />
-            <dl class="guides-section-container">
-              <% documents_by_section.each do |section| %>
-                <div class="guides-section">
-                  <dt><%= section['name'] %></dt>
+      <nav id="feature_nav"> <%# we use nav as an element to also provide the aria role and improve semantic meaning. %>
```

Comment:
This also feels more like a comment on the change than on the resulting code state

Reply:
Combination intent:
1. Education: The guides were all div-itis, so encouraging semantic markup wherever possible is an educational opportunity to developers who have minimal exposure to HTML & CSS.
2. ARIA note so we don't end up with redundant `role="navigation"` call here.

Reply:
Removed for now in 3632e453